### PR TITLE
ctb: Change getL2Output to getL2OutputAfter

### PR DESCRIPTION
**Pull request by maurelian**
**Description**

Renames `getL2Output` to `getL2OutputAfter`, which is semantically more descriptive, and could prevent future issues due to misunderstandings. 

This change may break external code, and so is posted here for consideration and discussion. 

**Tests**

Tests are updated.

### CODE REVIEW

**Review by clabby — CHANGES_REQUESTED** [review]
Looks good- will approve after the gas snapshot is updated.

File: packages/sdk/src/cross-chain-messenger.ts [hunk 1]
```diff
@@ -231,7 +231,7 @@ export class CrossChainMessenger {
    *
    * @returns The L2OutputOracle parameters.
    */
-  public async getL2OutputOracleParameters(): Promise<L2OutputOracleParameters> {
+  public async getL2OutputAfterOracleParameters(): Promise<L2OutputOracleParameters> {
```
Comment (smartcontracts) [inline packages/sdk/src/cross-chain-messenger.ts:234]: Lawl I think this is an errant find/replace

Comment (maurelian) [reply packages/sdk/src/cross-chain-messenger.ts:234]: barf

**Review by clabby — APPROVED** [review]

**Review by smartcontracts — APPROVED** [review]

**Review by smartcontracts — CHANGES_REQUESTED** [review]
Superseded by #4093

**Comment by tynes** [comment]
Going to close this in favor of #4093

**Pull request closed without merging**